tests/user-test.js

```diff
@@ -38,11 +39,11 @@ describe('User', () => {
       {
         "ingredient": 11547,
         "amount": 5
-      }];
+      }]
```

Comment:
I was unaware that it was not clean syntax to have the semi-colon at the end of that array?  Do you have a source for this?  Perhaps the linter?

Reply:
If you have a ; at the end of that it will throw an error when testing. I was not aware either, that was just the only way I could get NPM to run tests correctly.

Reply:
I do not have a source but when I ran NPM test it threw an error at that semicolon.

Reply:
Interesting, I'll ask one of the instructors if they can shed some light on why that's occurring.

Reply:
I figured out why it was throwing the error here: essentially, that array is actually within the parameters of the newly instantiated User.  I placed the semi-colon there because it seemed like it was the end of the line of code, but I forgot what I was working on.  Essentially my incorrect code looked like this: `user1 = new User(1, 'Boba', [{"ingredient": 1077, "amount": 1}, {"ingredient": 14412, "amount": 1}, {"ingredient": 1009054, "amount": 3}];)`

If you notice, at the very end of that code block there is my incorrectly placed semi-colon inside of the parameter parentheses.  So that's why NPM was throwing you an error.  I'm glad you caught that, it was a good learning experience for me to hunt down why exactly this was incorrect.